### PR TITLE
Use translate.localePicker.translateParams in AlternateHrefLangElemets component

### PR DESCRIPTION
**Pull request by patrick-durrer**

### CODE REVIEW

File: components/AlternateHrefLangElements.php [hunk 1]
```diff
@@ -3,6 +3,7 @@
 namespace Rainlab\Translate\Components;
 
 use Cms\Classes\ComponentBase;
+use Illuminate\Support\Facades\Event;
```
Comment (LukeTowers) [inline components/AlternateHrefLangElements.php]: Why do you fully specify that facade instead of just using `use Event`?

**Comment by alxy** [comment]
I think, if this is accepted, the event needs a more generic name, as the `localePicker` part of it doesnt make too much sense in this case. Maybe `translate.translateParams` is already okay.

**Comment by LukeTowers** [comment]
@alxy the event is already in use isn't it? Probably not going to change it now.

@patrick-durrer what is the purpose of this?

**Comment by mjauvin** [comment]
@LukeTowers @bennothommo this PR would potentially solve the Rainlab.Blog slug translation issues in AlternateHrefLangElements component with help from https://github.com/rainlab/blog-plugin/pull/436

**Comment by mjauvin** [comment]
I have tested this along with rainlab/blog-plugin#436 and I get the correct translated URLs in the added`<link rel="alternate" hreflang=...`  in <head> section of the page.

I would be inclined to merge this.

**Comment by mjauvin** [comment]
@patrick-durrer @tobias-kuendig could you apply @LukeTowers' requested fix to the facade?

**Pull request merged**